modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/StepsTable.scala

```diff
@@ -1023,6 +1032,24 @@ object StepsTable extends Columns {
     }.getOrEmpty
   }
 
+  private val computeScrollBarWidth: CallbackTo[Double] = {
+    import org.querki.jquery.$
+
+    ref.get.map(_.getDOMNode.toHtml).asCallback.map{
+      _.flatten.flatMap { tableNode =>
+        // Table has a Grid inside, which is the one actually showing the scroll bar.
+        $(tableNode).find(".ReactVirtualized__Table__Grid").get(0).toOption.map {
+          case gridNode: HTMLElement => gridNode.offsetWidth - gridNode.clientWidth
+          case _                     => 0.0
+        }
+      }.getOrElse(0.0)
+    }
+  }
+
+  def didUpdate(b: DidUpdate): Callback = {
+    computeScrollBarWidth >>= {sw => b.modState(State.scrollBarWidth.set(sw)) }
+  }
+
```

Comment:
TBH, this is a hack and I don't like it: we are relying on `virtualized`'s internal DOM structure.

However, it was the only way I could find to correctly and (hopefully) universally compensate for the scroll bar width.

It's worth noting that `virtualized` does know the scroll bar width: https://github.com/bvaughn/react-virtualized/blob/bc5ae2fa084e20323b445b31fd72acee90e0ab5f/source/Table/Table.js#L727 but it doesn't expose. Maybe we could submit a PR to them and live with this until it's published?

Reply:
Right, and I want to remove the jquery dependency. Shouldn't it be possible to call `_setScrollbarWidth` via `js.Dynamic`? I'm not totally sure. and we probably shouldn't spend too much time on this
Somehow I thought that there would be some DOM property that would give you that but I guess there isn't